health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/PopInboxTable.js

```
@@ -46,6 +986,35 @@ const PopInboxTable = ({ ...props }) => {
           return 0;
         },
       },
+      ...(
+        censusData?.[0]?.additionalFields
```

Biome

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)